lib/github-pages.rb

```diff
@@ -7,32 +7,31 @@ class GitHubPages
   def self.gems
     {
       # Jekyll
-      "jekyll"                    => "2.4.0",
-      "jekyll-coffeescript"       => "1.0.1",
+      "jekyll"                    => "3.0.1",
       "jekyll-sass-converter"     => "1.3.0",
 
       # Converters
       "kramdown"                  => "1.9.0",
-      "maruku"                    => "0.7.0",
       "rdiscount"                 => "2.1.8",
       "redcarpet"                 => "3.3.3",
       "RedCloth"                  => "4.2.9",
 
       # Liquid
-      "liquid"                    => "2.6.2",
+      "liquid"                    => "3.0.6",
 
       # Highlighters
-      "pygments.rb"               => "0.6.3",
+      "rouge"                     => "1.10.1",
 
       # Plugins
-      "jemoji"                    => "0.5.0",
-      "jekyll-mentions"           => "0.2.1",
-      "jekyll-redirect-from"      => "0.8.0",
+      "jemoji"                    => "0.5.1",
+      "jekyll-mentions"           => "1.0.0",
+      "jekyll-redirect-from"      => "0.9.0",
```

Comment:
0.9.1 released today. Should I make another PR?

https://github.com/jekyll/jekyll-redirect-from/releases/tag/v0.9.1

https://github.com/jekyll/jekyll-redirect-from/compare/v0.9.0...v0.9.1